cluster/gce/config-default.sh

```diff
@@ -407,6 +407,9 @@ SCHEDULING_ALGORITHM_PROVIDER="${SCHEDULING_ALGORITHM_PROVIDER:-}"
 # Optional: install a default StorageClass
 ENABLE_DEFAULT_STORAGE_CLASS="${ENABLE_DEFAULT_STORAGE_CLASS:-true}"
 
+# Optional: install volume snapshot CRDs
+ENABLE_SNAPSHOTS_CRDS="${ENABLE_SNAPSHOTS_CRDS:-true}"
```

Comment:
It's not just the CRDs now, it's also the controller right? Maybe just "ENABLE_VOLUME_SNAPSHOTS"

Reply:
done